lib/stack.rb

```diff
@@ -1,3 +1,63 @@
+require_relative './node'
```

Comment:
Imagine a scenario where we can add any kind of data-structure in order to test the performance of the stack.